app/src/main/java/org/wikipedia/views/SurveyDialog.kt

```diff
@@ -81,13 +96,13 @@ object SurveyDialog {
 
     private fun showFeedbackSnackbarAndTooltip(activity: Activity, messageId: Int, source: Constants.InvokeSource) {
         FeedbackUtil.showMessage(activity, messageId)
-        sendAnalyticsEvent("feedback_submit_toast", "feedback_form", source)
         when (source) {
             Constants.InvokeSource.SUGGESTED_EDITS_RECENT_EDITS -> {
+                PatrollerExperienceEvent.logAction("feedback_submit_toast", "feedback_form", "")
```

Comment:
We can ignore `""` in the last parameter.

Reply:
OK!